source/containers.tex

```diff
@@ -69,12 +69,12 @@
 aligned buffers and call \tcode{construct} to place the element into the buffer.
 \end{note}
 
-\rSec2[container.gen.reqmts]{General containers}
```

Comment:
If the stable label already existed in C++23, we should add an entry to the stable label changes.

Reply:
So add this to xrefdeltas.tex?

```
\movedxref{container.requirements.general}{container.intro.reqmts}
\movedxref{container.gen.reqmts}{container.requirements.general}
```

Reply:
Yes, that's right!

Reply:
OK, added and rebased

Reply:
Continue rebasing until morale improves.

Reply:
incorrect entries in xrefdelta.tex:
spurious container.requirements.general